examples/treegrid/css/treegrid-1.css

```diff
@@ -91,7 +91,8 @@
   cursor: pointer;
 
   /* Load both right away so there is no lag when we need the other */
-  background-image: url("expand-icon.svg"), url("expand-icon-highlighted.svg");
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='12' height='12'%3E%3Cpolygon fill='black' points='2,0 2,10 10,5'%3E%3C/polygon%3E%3C/svg%3E%0A"),
```

Comment:
Oops, I think my assumption about the trailing `;` in the cspell ignore `"url\\(\"data\\:image/svg\\+xml.*\"\\);"` is bad 😦

Reply:
@nschonni 
Heh, yeah - apparently it can be a comma, too... :)
https://developer.mozilla.org/en-US/docs/Web/CSS/background-image